tools/obscuroscan/obscuroscan.go

```diff
@@ -544,3 +556,14 @@ func logAndSendErr(resp http.ResponseWriter, msg string) {
 	fmt.Println(msg)
 	http.Error(resp, msg, httpCodeErr)
 }
+
+// Enables CORS to allow Obscuroscan API to serve other web apps. Returns true if the request was a pre-flight, e.g. OPTIONS, to stop further processing.
+func (o *Obscuroscan) enableCORS(resp http.ResponseWriter, req *http.Request) bool {
```

Comment:
Probably better to re-use it rather than duplicate it across from the wallet extension code if possible (don't think there's anything specific to Obscuroscan here).

I'd suggest adding `EnableCORS(...)` and the block of constants it uses somewhere like `go/common/httputil/httputil.go` so that both tools can use it.

Reply:
thanks